### ISSUE TITLE
Do these lines need to be deleted?

### ISSUE THREAD

**Issue by DAnn2012** [nolimits4web/Dom7#49]
@nolimits4web Do these lines need to be deleted?

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/src/%24.js#L46

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/src/scroll.js#L92

Thanks.

**Comment by nolimits4web** [comment]
Not necessary as these comment don't affect anything

**Comment by DAnn2012** [comment]
@nolimits4web I understand, but it was because in this way they are also reported in this files:

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/package/dom7.js#L379

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/package/dom7.js#L1612

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/package/dom7.esm.js#L221

https://github.com/nolimits4web/dom7/blob/925718340f89d480c04fa50a86844822b534099c/package/dom7.esm.js#L1394

and also in this framework7 files (Lines 393-1562):

https://unpkg.com/browse/framework7@6.1.0-beta.6/framework7-bundle.js

https://unpkg.com/browse/framework7@6.1.0-beta.6/framework7.js

Thanks.

**Comment by nolimits4web** [comment]
Yes, but anyway it is still doesn't affect anything :) PR is welcome if it bothers you
